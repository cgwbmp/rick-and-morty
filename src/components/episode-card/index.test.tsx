import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import EpisodeCard from '.';

describe('<EpisodeCard />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<EpisodeCard name="Name" episode="01" />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('DOM structure not changed', () => {
    const component = renderer.create(<EpisodeCard name="Name" episode="01" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
