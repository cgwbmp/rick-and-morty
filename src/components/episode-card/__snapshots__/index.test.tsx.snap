// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EpisodeCard /> DOM structure not changed 1`] = `
<div
  className="card"
>
  <p
    className="info"
  >
    01
  </p>
  <h4
    className="title"
  >
    Name
  </h4>
</div>
`;
